reporter/src/test/kotlin/utils/FreeMarkerTemplateProcessorTest.kt

```diff
@@ -183,35 +215,76 @@ class FreeMarkerTemplateProcessorTest : WordSpec({
         }
     }
 
-    "mergeResolvedLicenses" should {
-        "merge the original expressions by license sources" {
-            val resolvedLicenses = listOf(
-                ResolvedLicense(
-                    license = "MIT".toSpdx() as SpdxSingleLicenseExpression,
-                    originalDeclaredLicenses = emptySet(),
-                    originalExpressions = mapOf(LicenseSource.DECLARED to setOf("MIT".toSpdx())),
-                    locations = emptySet()
-                ),
-                ResolvedLicense(
-                    license = "MIT".toSpdx() as SpdxSingleLicenseExpression,
-                    originalDeclaredLicenses = emptySet(),
-                    originalExpressions = mapOf(LicenseSource.DECLARED to setOf("GPL-2.0-only OR MIT".toSpdx())),
-                    locations = emptySet()
-                )
+    "mergeLicenses" should {
+        "merge the expressions from packages by license sources" {
+            val projects = ORT_RESULT.getProjects().toList()
+            val resolver = mockk<LicenseInfoResolver>()
+            expectResolveLicenseInfo(
+                resolver,
+                projects[0].id,
+                "MIT",
+                mapOf(LicenseSource.DECLARED to setOf("MIT".toSpdx()))
+            )
```

Comment:
Nit: Generally our code style demands blank lines before and after code blocks.

Reply:
Similar below.

Reply:
Blank lines were added before and after the `expectResolveLicenseInfo()` invocations, but multiple invocations in series were kept together.